_utilities/layout-grid.md

```diff
@@ -504,9 +504,9 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
 When generating your CSS from USWDS source files, you have the option of customizing many system defaults by modifying project theme variables. USWDS also provides grid mixins for adding grid functionality to custom semantic component CSS.
 
 ### Variables
-Variables and maps determine the number of columns, gutter width, and media-query point at which to begin floating columns. We use variables to generate the predefined grid classes documented on this page, as well as for the custom mixins noted under USWDS-theme-spacing.scss and USWDS-theme-utilities.scss.
+Variables and maps determine the number of columns, gutter width, and media-query point at which to begin floating columns. We use variables to generate the predefined grid classes documented on this page, as well as for the custom mixins noted under [Spacing settings](#spacing-settings) and [Utilities settings](#utilities-settings)
```

Comment:
Missing period at end of sentence.

![image](https://user-images.githubusercontent.com/3385219/200364918-2a4069e2-ebdc-4cda-b65c-e3acd358eb2f.png)